deploy/docker/notebook/base_gpu.yaml

```diff
@@ -2,10 +2,19 @@ name: base_gpu
 base: "labshare/polus-cuda-base:focal-20210416-cuda-11-3@sha256:92c5f7c88f1cc2661a2a652733ee766f41c03b2d9a33b206c3d47d6bf81a6517"
```

Comment:
base and base_gpu seem very similar.  I usually like to follow DRY (Do not repeat yourself).  is there anyway we can use a base template and just add the stuff that gpu needs.  Different docker file and some of the GPU requirements"

Minor issue but it could become a problem for support when we want to use gpu more and we have to keep the versions consistent between the two.

Reply:
Reading these makes me nervous that there may be differences in the long code blocks so it would be nice to have a clear distinction between what we need for GPU and what we need for a normal deployment.

Reply:
Yeah, everything below the base image and name in these 2 templates is copy-pasted every time I make changes. `railyard` actually allows you to merge the yaml definitions before making template substitutions, so I can make 3 yamls:

base.yaml
```
name: base
base: "ubuntu:focal-20210416@sha256:86ac87f73641c920fb42cc9612d4fb57b5626b56ea2a19b894d0673fd5b4f2e9"
```

base_gpu.yaml
```
name: base_gpu
base: "labshare/polus-cuda-base:focal-20210416-cuda-11-3@sha256:92c5f7c88f1cc2661a2a652733ee766f41c03b2d9a33b206c3d47d6bf81a6517"
```

main.yaml
```
env:
  - HOME: /home/$NB_USER
  - MINICONDA_VERSION: py39_4.10.3
  - MINICONDA_INSTALLER_HASHSUM: 8c69f65a4ae27fb41df0fe552b4a8a3b
...
```

This will satisfy DRY and be less prone to errors

Reply:
Fixed